assets/js/base/components/cart-checkout/shipping-rates-control/index.tsx

```diff
@@ -81,6 +81,7 @@ interface ShippingRatesControlProps {
 	shippingRatesLoading: boolean;
 	noResultsMessage: ReactElement;
 	renderOption: PackageRateRenderOption;
+	context: 'wc/cart' | 'wc/checkout';
```

Comment:
Should we make this match the block name? `woocommerce/cart` and `woocommerce/checkout`.

Reply:
I am ok with matching with the block name, that would be ideal. But I went with the `wc/` prefix because this was already used in the project for the same context purpose. I brought this up for discussion internally p1645094942745539-slack-C8X6Q7XQU 

 I am afraid we would be introducing yet another value besides `wc/cart` and `cart`; if would want to normalize we would have to deprecate the other ones. 
 
 With this in mind, how do you think we should move fw?

Reply:
Message context confuses me because we give messages a certain context/area but render them in different places. The current context values don't really indicate anything about placement. Block name is closer, but still, do we have **multiple contexts within a block** to worry about if we changed that? I think we may - payment methods have a context too I believe.